src/dirnode.cpp

```diff
@@ -573,7 +579,17 @@ float RDirNode::getArea() const{
 
 void RDirNode::calcRadius() {
 
-    float total_file_area = file_area * visible_count;
+    float total_file_area = 0;
+    if (gGourceSettings.scale_by_file_size) {
+        for (RFile* f : files) {
+            if (!f->isHidden()) {
+                float r = f->getSize() / 2.0f;
```

Comment:
So this sets the radius to file size. It means the actual dot size will be proportional to file size squared. The area of the dots **will not be proportional** to file size.